clang/include/clang/Driver/Options.td

```diff
@@ -2955,6 +2955,10 @@ defm clangir_direct_lowering : BoolFOption<"clangir-direct-lowering",
   FrontendOpts<"ClangIRDirectLowering">, DefaultTrue,
   PosFlag<SetTrue, [], [ClangOption, CC1Option], "Lower directly from ClangIR to LLVM">,
   NegFlag<SetFalse, [], [ClangOption, CC1Option],  "Lower through MLIR to LLVM">>;
+defm clangir_analysis_only_pipeline : BoolFOption<"clangir-analysis-only",
+  FrontendOpts<"ClangIRAnalysisOnlyPipeline">, DefaultFalse,
+  PosFlag<SetTrue, [], [ClangOption, CC1Option], "Enable CIR analysis for the traditional code gen pipeline">,
```

Comment:
Rephase a bit: `Enable CIR analysis but keep traditional LLVM codegen (not through CIR)`

Reply:
Done